projects/core/src/auth/user-auth/services/auth-http-header.service.ts

```diff
@@ -273,6 +273,7 @@ export class AuthHttpHeaderService implements OnDestroy {
           // we want to refresh the access token only when it is old.
           // this is a guard for the case when there are multiple parallel http calls
           if (
+            token &&
```

Comment:
What is the purpose of adding `token &&` ?

Reply:
discussion results: This results in a logical change, so we will defer this to investigation for 7.0 release.  For now, we will do a type casting.